sdcm/utils/common.py

```diff
@@ -979,6 +979,25 @@ def tag_ami(ami_id, tags_dict, region_name):
     LOGGER.info("tagged %s with %s", ami_id, tags)
 
 
+def get_compact_tables(loader_node, db_node, ks, request_timeout=300, non_compact=False):
```

Comment:
may be better rename to get_tables_with_compaction?

Reply:
Fixed

Reply:
Fixed